test/remote/gateways/remote_mit_test.rb

```diff
@@ -117,7 +117,7 @@ def test_failed_refund
 
   def test_transcript_scrubbing
     transcript = capture_transcript(@gateway) do
-      @gateway.purchase(@amount, @credit_card, @options_success)
+      @gateway.authorize(@amount, @credit_card, @options_success)
```

Comment:
Is there a reason why this was changed from purchase to authorize?

Reply:
This one is a mistake on our part and we are returning it to the previous version.